.travis.yml

```diff
@@ -8,97 +8,72 @@ cache:
-    - php: 7.1
-      env: DB=pgsql MOODLE_BRANCH=MOODLE_27_STABLE_FSAPI MOODLE_REPO=https://github.com/kenneth-hendricks/moodle-fs-api.git
+      env: DB=mysqli MOODLE_BRANCH=MOODLE_35_STABLE
     - php: 7.1
-      env: DB=mysqli MOODLE_BRANCH=MOODLE_27_STABLE_FSAPI MOODLE_REPO=https://github.com/kenneth-hendricks/moodle-fs-api.git
+      env: DB=pgsql  MOODLE_BRANCH=MOODLE_35_STABLE
     - php: 7.0
-      env: DB=pgsql MOODLE_BRANCH=MOODLE_27_STABLE_FSAPI MOODLE_REPO=https://github.com/kenneth-hendricks/moodle-fs-api.git
+      env: DB=mysqli MOODLE_BRANCH=MOODLE_35_STABLE
     - php: 7.0
-      env: DB=mysqli MOODLE_BRANCH=MOODLE_27_STABLE_FSAPI MOODLE_REPO=https://github.com/kenneth-hendricks/moodle-fs-api.git
+      env: DB=pgsql  MOODLE_BRANCH=MOODLE_35_STABLE
+    - php: 7.2
+      env: DB=mysqli MOODLE_BRANCH=MOODLE_36_STABLE
+    - php: 7.2
+      env: DB=pgsql  MOODLE_BRANCH=MOODLE_36_STABLE
+    - php: 7.1
+      env: DB=mysqli MOODLE_BRANCH=MOODLE_36_STABLE
+    - php: 7.1
+      env: DB=pgsql  MOODLE_BRANCH=MOODLE_36_STABLE
     - php: 7.0
-      env: DB=pgsql MOODLE_BRANCH=MOODLE_37_STABLE
+      env: DB=mysqli MOODLE_BRANCH=MOODLE_36_STABLE
     - php: 7.0
+      env: DB=pgsql  MOODLE_BRANCH=MOODLE_36_STABLE
+    - php: 7.2
       env: DB=mysqli MOODLE_BRANCH=MOODLE_37_STABLE
-    - php: 7.0
-      env: DB=pgsql MOODLE_BRANCH=MOODLE_38_STABLE
-    - php: 7.0
-      env: DB=mysqli MOODLE_BRANCH=MOODLE_38_STABLE
-    - php: 7.0
-      env: DB=pgsql MOODLE_BRANCH=master
-    - php: 7.0
-      env: DB=mysqli MOODLE_BRANCH=master
-    - php: 5.6
-      env: DB=pgsql MOODLE_BRANCH=MOODLE_34_STABLE
-    - php: 5.6
-      env: DB=pgsql MOODLE_BRANCH=MOODLE_35_STABLE
-    - php: 5.6
-      env: DB=pgsql MOODLE_BRANCH=MOODLE_36_STABLE
-    - php: 5.6
-      env: DB=mysqli MOODLE_BRANCH=MOODLE_36_STABLE
-    - php: 5.6
-      env: DB=pgsql MOODLE_BRANCH=MOODLE_37_STABLE
-    - php: 5.6
+    - php: 7.2
+      env: DB=pgsql  MOODLE_BRANCH=MOODLE_37_STABLE
+    - php: 7.1
       env: DB=mysqli MOODLE_BRANCH=MOODLE_37_STABLE
-    - php: 5.6
-      env: DB=pgsql MOODLE_BRANCH=MOODLE_38_STABLE
-    - php: 5.6
+    - php: 7.1
+      env: DB=pgsql  MOODLE_BRANCH=MOODLE_37_STABLE
+    - php: 7.2
       env: DB=mysqli MOODLE_BRANCH=MOODLE_38_STABLE
-    - php: 5.6
-      env: DB=pgsql MOODLE_BRANCH=master
-    - php: 5.6
-      env: DB=mysqli MOODLE_BRANCH=MOODLE_34_STABLE
-    - php: 5.6
-      env: DB=mysqli MOODLE_BRANCH=MOODLE_35_STABLE
-    - php: 5.6
-      env: DB=mysqli MOODLE_BRANCH=master
+    - php: 7.2
+      env: DB=pgsql  MOODLE_BRANCH=MOODLE_38_STABLE
+    - php: 7.1
+      env: DB=mysqli MOODLE_BRANCH=MOODLE_38_STABLE
+    - php: 7.1
+      env: DB=pgsql  MOODLE_BRANCH=MOODLE_38_STABLE
 
 before_install:
+  - export IGNORE_PATHS=classes/azure/StreamWrapper.php,classes/local/store/azure/stream_wrapper.php,classes/local/store/swift/stream_wrapper.php
```

Comment:
What is this for?

Reply:
Added `admin/tool/objectfs/thirdpartylibs.xml` instead